FPGA1394_QLA/ipcore_dir/FIR.xco

```diff
@@ -0,0 +1,112 @@
+##############################################################
+#
+# Xilinx Core Generator version 14.7
+# Date: Tue May 18 16:19:53 2021
+#
+##############################################################
+#
+#  This file contains the customisation parameters for a
+#  Xilinx CORE Generator IP GUI. It is strongly recommended
+#  that you do not manually alter this file as it may cause
+#  unexpected and unsupported behavior.
+#
+##############################################################
+#
+#  Generated from component: xilinx.com:ip:fir_compiler:6.3
+#
+##############################################################
+#
+# BEGIN Project Options
+SET addpads = false
+SET asysymbol = true
+SET busformat = BusFormatAngleBracketNotRipped
+SET createndf = false
+SET designentry = Verilog
+SET device = xc6slx45
+SET devicefamily = spartan6
+SET flowvendor = Other
+SET formalverification = false
+SET foundationsym = false
+SET implementationfiletype = Ngc
+SET package = fgg484
+SET removerpms = false
+SET simulationfiles = Behavioral
+SET speedgrade = -2
+SET verilogsim = true
+SET vhdlsim = false
+# END Project Options
+# BEGIN Select
+SELECT FIR_Compiler xilinx.com:ip:fir_compiler:6.3
+# END Select
+# BEGIN Parameters
+CSET bestprecision=true
+CSET channel_sequence=Basic
+CSET clock_frequency=6.144
+CSET coefficient_buffer_type=Automatic
+CSET coefficient_file=no_coe_file_loaded
+CSET coefficient_fractional_bits=20
+CSET coefficient_reload=false
+CSET coefficient_sets=1
+CSET coefficient_sign=Unsigned
+CSET coefficient_structure=Inferred
+CSET coefficient_width=16
+CSET coefficientsource=Vector
+CSET coefficientvector=0.014308,0.017565,0.02092,0.024275,0.027728,0.030985,0.034241,0.037201,0.039964,0.042431,0.044504,0.046181,0.047365,0.048155,0.048352,0.048155,0.047365,0.046181,0.044504,0.042431,0.039964,0.037201,0.034241,0.030985,0.027728,0.024275,0.02092,0.017565,0.014308
```

Comment:
The coefficient vector is hard-coded.  Is it possible to reconfigure this, for example by downloading new values?  I believe you have 29 coefficients which, although shown as floating point values here, are eventually converted to 16-bit coefficients.  So, we could do a block write of coefficients.  I see above a `CSET coefficient_reload=false`, so probably that would need to be changed. What else would be needed?